tests/test_edge_cases.py

```diff
@@ -30,7 +30,7 @@ class Model(BaseModel):
     assert repr(m.__fields__['v']) == "ModelField(name='v', type=Union[str, bytes], required=True)"
 
     m = Model(v=b'b')
-    assert m.v == 'b'
+    assert m.v == b'b'
```

Comment:
This fixes also some edge cases. I just fixed the tests here but I guess we could remove them to avoid unnecessary tests